### PR TITLE
[Fixes #223] Fix standalone tests to actually publish standalone app on CoreClr

### LINKED ISSUE #223: Fix standalone tests to actually publish standalone app on CoreClr
Currently we do not run standalone tests for CoreClr. The following condition doesn't make much sense as we are testing either Standalone/Portable in the context of CoreClr
https://github.com/aspnet/Entropy/blob/dev/test/Entropy.FunctionalTests/TestServices.cs#L53 

cc @Eilon @ryanbrandenburg

### PR DESCRIPTION
**Pull request by kichalla**
…on CoreClr

### CODE REVIEW

File: test/Entropy.FunctionalTests/E2ETestBase.cs [hunk 1]
```diff
@@ -25,74 +25,43 @@ protected E2ETestBase(ITestOutputHelper output, string siteName) : base(output)
...
+        [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Portable)]
+        [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Standalone)]
+        [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x86, ApplicationType.Standalone,
+            Skip = "https://github.com/aspnet/Hosting/issues/949")]
+        [InlineData(ServerType.WebListener, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Portable)]
+        [InlineData(ServerType.WebListener, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Standalone)]
+        [InlineData(ServerType.WebListener, RuntimeFlavor.CoreClr, RuntimeArchitecture.x86, ApplicationType.Standalone,
+            Skip = "https://github.com/aspnet/Hosting/issues/949")]
+        [InlineData(ServerType.Kestrel, RuntimeFlavor.Clr, RuntimeArchitecture.x64, ApplicationType.Standalone)]
+        [InlineData(ServerType.WebListener, RuntimeFlavor.Clr, RuntimeArchitecture.x64, ApplicationType.Standalone)]
```
Comment (kichalla) [inline test/Entropy.FunctionalTests/E2ETestBase.cs]: There is a gap in the api of enum ApplicationType. We should probably add a value like `None` to it as for desktop it does not make sense to explicitly specify standalone.

**Comment by kichalla** [comment]
ping

File: test/Entropy.FunctionalTests/TestServices.cs [hunk 2]
```diff
@@ -39,6 +39,7 @@ public static void LogResponseOnFailedAssert(this ILogger logger, HttpResponseMe
             ServerType serverType,
             RuntimeFlavor runtimeFlavor,
             RuntimeArchitecture architecture,
+            ApplicationType applicationType,
```
Comment (JunTaoLuo) [inline test/Entropy.FunctionalTests/TestServices.cs]: Why is this parameter unused?

File: test/Entropy.FunctionalTests/E2ETestBase.cs [hunk 3]
```diff
@@ -25,74 +25,43 @@ protected E2ETestBase(ITestOutputHelper output, string siteName) : base(output)
...
-        public Task WebListenerX86CLRWindows()
+        [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Portable)]
+        [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Standalone)]
+        [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x86, ApplicationType.Standalone,
+            Skip = "https://github.com/aspnet/Hosting/issues/949")]
+        [InlineData(ServerType.WebListener, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Portable)]
+        [InlineData(ServerType.WebListener, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, ApplicationType.Standalone)]
+        [InlineData(ServerType.WebListener, RuntimeFlavor.CoreClr, RuntimeArchitecture.x86, ApplicationType.Standalone,
+            Skip = "https://github.com/aspnet/Hosting/issues/949")]
+        [InlineData(ServerType.Kestrel, RuntimeFlavor.Clr, RuntimeArchitecture.x64, ApplicationType.Standalone)]
```
Comment (JunTaoLuo) [inline test/Entropy.FunctionalTests/E2ETestBase.cs]: Kestrel Clr x86? it's being skipped right now but we should add it.

**Review by JunTaoLuo — APPROVED** [review]

**Comment by kichalla** [comment]
aa5aca144357e7bbf9ed820df27fc11164d083a4

**Pull request closed without merging**